src/main/kotlin/dev/mfazio/wwc2023/types/domain/Team.kt

```diff
@@ -7,73 +7,73 @@ sealed class Team(
-    object Brazil : Team("BRA", "Brazil", "BR", "G")
-    object Cameroon : Team("CMR", "Cameroon", "CM", "G")
-    object Canada : Team("CAN", "Canada", "CA", "F")
-    object CostaRica : Team("CRC", "Costa Rica", "CR", "E")
-    object Croatia : Team("CRO", "Croatia", "HR", "F")
+    object Argentina : Team("ARG", "Argentina", "AR", "G")
+    object Australia : Team("AUS", "Australia", "AU", "B")
+    object Brazil : Team("BRA", "Brazil", "BR", "F")
+    object Canada : Team("CAN", "Canada", "CA", "B")
+    object ChinaPR : Team("CHN", "China PR", "CN", "D")
+    object Colombia : Team("COL", "Colombia", "CO", "H")
+    object CostaRica : Team("CRC", "Costa Rica", "CR", "C")
     object Denmark : Team("DEN", "Denmark", "DK", "D")
-    object Ecuador : Team("ECU", "Ecuador", "EC", "A")
-    object England : Team("ENG", "England", "EN", "B")
-    object France : Team("FRA", "France", "FR", "D")
-    object Germany : Team("GER", "Germany", "DE", "E")
-    object Ghana : Team("GHA", "Ghana", "GH", "H")
-    object Iran : Team("IRN", "IR Iran", "IR", "B")
-    object Japan : Team("JPN", "Japan", "JP", "E")
+    object England : Team("ENG", "England", "GB", "D")
+    object France : Team("FRA", "France", "FR", "F")
+    object Germany : Team("GER", "Germany", "DE", "H")
+    object Haiti : Team("HAI", "Haiti", "HT", "D")
+    object Italy : Team("ITA", "Italy", "IT", "G")
+    object Jamaica : Team("JAM", "Jamaica", "JM", "F")
+    object Japan : Team("JPN", "Japan", "JP", "C")
     object KoreaRepublic : Team("KOR", "Korea Republic", "KR", "H")
-    object Mexico : Team("MEX", "Mexico", "MX", "C")
-    object Morocco : Team("MAR", "Morocco", "MA", "F")
-    object Netherlands : Team("NED", "Netherlands", "NL", "A")
-    object Poland : Team("POL", "Poland", "PL", "C")
-    object Portugal : Team("POR", "Portugal", "PT", "H")
-    object Qatar : Team("QAT", "Qatar", "QA", "A")
-    object SaudiArabia : Team("KSA", "Saudi Arabia", "SA", "C")
-    object Senegal : Team("SEN", "Senegal", "SN", "A")
-    object Serbia : Team("SRB", "Serbia", "RS", "G")
-    object Spain : Team("ESP", "Spain", "ES", "E")
-    object Switzerland : Team("SUI", "Switzerland", "CH", "G")
-    object Tunisia : Team("TUN", "Tunisia", "TN", "D")
-    object Uruguay : Team("URU", "Uruguay", "UY", "H")
-    object USA : Team("USA", "USA", "US", "B")
-    object Wales : Team("WAL", "Wales", "WA", "B")
+    object Morocco : Team("MAR", "Morocco", "MA", "H")
+    object Netherlands : Team("NED", "Netherlands", "NL", "E")
+    object NewZealand : Team("NZL", "New Zealand", "NZ", "A")
+    object Nigeria : Team("NGA", "Nigeria", "NG", "B")
+    object Norway : Team("NOR", "Norway", "NO", "A")
+    object Panama : Team("null", "Panama", "PA", "F")
+    object Philippines : Team("PHI", "Philippines", "PH", "A")
+    object Portugal : Team("POR", "Portugal", "PT", "E")
+    object RepublicOfIreland : Team("IRL", "Republic of Ireland", "IE", "B")
```

Comment:
Another one where we can shorten things. I think `Ireland` is fine here. The order can be changed in that case, too.
```suggestion
    object fIreland : Team("IRL", "Ireland", "IE", "B")
```